commit aa975a3cda72f222c8224395bea8be20204036b1
Author: Pierre Rineau <pierre.rineau@makina-corpus.com>
Date:   Tue Sep 19 13:04:44 2017 +0200

    prevent PHP warnings to break rendering when working with Symfony in dev/debug mode

diff --git a/src/FrameReflower/Block.php b/src/FrameReflower/Block.php
index 5d9d1d2..957c1f6 100644
--- a/src/FrameReflower/Block.php
+++ b/src/FrameReflower/Block.php
@@ -591,7 +591,7 @@ class Block extends AbstractFrameReflower
                         continue;
                     }
                     $frameBox = $frame->get_frame()->get_border_box();
-                    $imageHeightDiff = $height * 0.8 - $frameBox['h'];
+                    $imageHeightDiff = $height * 0.8 - (int)$frameBox['h'];
 
                     $align = $frame->get_style()->vertical_align;
                     if (in_array($align, Style::$vertical_align_keywords) === true) {
